app/interface/deps/redis_deps.py

```
@@ -0,0 +1,9 @@
+from fastapi import Request
+from redis.asyncio import Redis
+
+
+async def get_redis_client(request: Request) -> Redis:
+    redis = getattr(request.app.state, "redis", None)
+    if redis is None:
+        raise RuntimeError("Redis에 연결되지 않았어요.")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)